Makefile-tests.am

```diff
@@ -121,6 +121,7 @@ _installed_or_uninstalled_test_scripts += \
 	tests/test-refs-collections.sh \
 	tests/test-remote-add-collections.sh \
 	tests/test-summary-collections.sh \
+	tests/test-pull-collections.sh \
```

Comment:
Can you squash tests in with the commit that introduced the functionality?  Not sure why one would want them separate.